Source/Extensions/Blazorise.DataGrid/DataGridColumn.cs

```diff
@@ -647,6 +762,31 @@ public SortDirection CurrentSortDirection
     /// </summary>
     [Parameter] public TextAlignment TextAlignment { get; set; }
 
+    /// <summary>
+    /// Gets or sets the text transformation for display cell.

```

Comment:
The doc mentions this is for **display cell.** However, both Filter and Aggregate parameters will fallback to these regular parameters. Is this clear to the user? I think it might be misleading as it says **"for display cell"**

Reply:
You're right. I will change a description a bit.